### PR TITLE
Update KubeCon NA 2022 videos

### PR DESCRIPTION
**Pull request by vanessaabankwah**
@staceypotter 

Signed-off-by: Vanessa Abankwah <96237690+vanessaabankwah@users.noreply.github.com>

### CODE REVIEW

File: content/en/resources.yaml [hunk 1]
```diff
@@ -17,10 +17,10 @@ resources:
   - youtube: h8G3LM9uIHk
     title: "Toward Full Adoption of GitOps and Best Practices at RingCentral - Ivan Anisimov & Tamao Nakahara"
     date: 2022-10-25
-  - youtube: 7dOKxIYEtvY
+  - youtube: EXp2xAbII_k
     title: "How to Achieve (Actual) GitOps with Terraform and Flux - Priyanka Ravi, Weaveworks"
     date: 2022-10-25 
-  - url: https://youtube.com/playlist?list=PL9lTuCFNLaD0NVkR17tno4X6BkxsbZZfr
+  - url:https://youtu.be/EXp2xAbII_k
```
Comment (dholbach) [inline content/en/resources.yaml:23]: Not sure this is correct. It replaces the GitOps Days playlist with RingCentral talk above.

Comment (vanessaabankwah) [reply content/en/resources.yaml:23]: Thanks I was editing during the meeting

**Review by vanessaabankwah — COMMENTED** [review]
Adding more videos

File: content/en/resources.yaml [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+    title: "Flux Maturity, Feature, and Contrib Update - Kingdon Barrett & Somtochi Onyekwere, Weaveworks"
+    date: 2022-10-27
+ - youtube: 9Ag45POgnKw
+    title: "Flagger, Linkerd, And Gateway API: Oh My! - Jason Morgan, Buoyant & Sanskar Jaiswal, Weaveworks"
+    date: 2022-10-26
+ - youtube: G1Mt4KE4Dao
     title: "Flux + Observability: Featuring Prometheus Operator and Pixie - Somtochi Onyekwere, Weaveworks"
     date: 2022-10-25
-  - youtube: Ums3Q9kMPd8
+ - youtube: Ums3Q9kMPd8
```
Comment (dholbach) [inline content/en/resources.yaml:11]:
```suggestion
  - youtube: Ums3Q9kMPd8
```

File: content/en/resources.yaml [hunk 3]
```diff
@@ -1,8 +1,14 @@
 resources:
-  - youtube: G1Mt4KE4Dao
+- youtube: PhV5dJtTaDw
+    title: "Flux Maturity, Feature, and Contrib Update - Kingdon Barrett & Somtochi Onyekwere, Weaveworks"
+    date: 2022-10-27
+ - youtube: 9Ag45POgnKw
+    title: "Flagger, Linkerd, And Gateway API: Oh My! - Jason Morgan, Buoyant & Sanskar Jaiswal, Weaveworks"
+    date: 2022-10-26
+ - youtube: G1Mt4KE4Dao
```
Comment (dholbach) [inline content/en/resources.yaml:8]:
```suggestion
  - youtube: PhV5dJtTaDw
    title: "Flux Maturity, Feature, and Contrib Update - Kingdon Barrett & Somtochi Onyekwere, Weaveworks"
    date: 2022-10-27
  - youtube: 9Ag45POgnKw
    title: "Flagger, Linkerd, And Gateway API: Oh My! - Jason Morgan, Buoyant & Sanskar Jaiswal, Weaveworks"
    date: 2022-10-26
  - youtube: G1Mt4KE4Dao
```

**Review by dholbach — APPROVED** [review]

**Pull request merged**